### PR TITLE
U/yoachim/satellites

### PR DESCRIPTION
**Pull request by yoachim**
Some tools for looking at the impact of satellite mega constellations. Not sure how to get the CI to pick up the new requirements, tried adding them to `test-requirements.txt` and `requirements.txt` but they don't get picked up.

### CODE REVIEW

**Comment by rhiannonlynne** [comment]
I'm not sure how the lsst-ts Jenkins requirements are set. It's possible it's using our requirements.txt file, but rather a proper conda "meta.yaml" recipe, so it would have to be updated there. (note that you will also have to update the meta.yaml in OUR condo-forge version of the feedstock).

File: tests/satellite_constellations/test_satellites.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    starlink_tles_v2,
+    oneweb_tles,
+    Constellation,
+)
+
+
+class TestSatellites(unittest.TestCase):
+    def test_constellations(self):
+        """Test instantiation of slicer sets slicer type as expected."""
+
```
Comment (rhiannonlynne) [inline tests/satellite_constellations/test_satellites.py:15]: Please update the docstring.

**Comment by rhiannonlynne** [comment]
I was going to ask why you would subclass the model_observatory instead of adding satellite information into the original satellite observatory (and leaving the utilities to feed the conditions objects, etc. here). But I suspect it's because it's not clear whether satellites need to be included in the model_observatory or not?

**Comment by yoachim** [comment]
Yeah, since this is one potential way of dodging satellites (and not really worth it imo), I figured it was better to not put the updates in the regular `ModelObservatory`. Plus, I like this as an example of how one can add information to the telemetry stream in the conditions object without needing to fork/branch/edit `rubin_sim` code.

**Comment by yoachim** [comment]
Someone should put links/instructions to the conda packaging system(s) in the README file.

**Pull request merged**